slides/resilient-component-libraries.mdx

```diff
@@ -1125,19 +1202,63 @@ const RedBox = ({className, ...props}) => (
 
 ---
 
-<Figure
-	src={'/assets/component-driven.svg'}
-	alt="Component-driven Logo"
-	caption={
-		<a href="https://component-driven.io">https://component-driven.io</a>
-	}
-/>
+<Primary>
+
+# React.Context
+
+</Primary>
+
+---
+
+> Context is designed to share data that can be considered “global” for a tree of React components
+
+---
+
+<SplitTitle>
+
+## Avoid "prop drilling"
+
+```jsx
+import { ThemeProvider } from 'styled-components';
+import theme from '../theme';
+
+<ThemeProvider theme={theme}>
+	<App />
+</ThemeProvider>;
+```
+
+</SplitTitle>
+
+---
+
+## Avoid re-renders by passing React reconciliation
```

Comment:
Bypassing? I'm not sure what “passing React reconciliation” means.

Reply:
yes! thanks!